.github/workflows/publish-1.0-docs-nightly.yml

```diff
@@ -0,0 +1,76 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Based on Apache Arrow's java-nightly workflow
+# https://github.com/apache/arrow/blob/master/.github/workflows/java_nightly.yml
+name: Publish nightly 1.0 documentation
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
If we are publishing docs for 1.0 it shouldn't be done via a cron job

Reply:
I can live with getting rid of the cron schedule. Right now, it is more convenient to run it daily.

There would be no plan to run it daily indefinitely - ie the cron schedule could be removed later.

Reply:
We can run it on `push` for the final docs branch. In the meantime, it might be enough to run it on `workflow_dispatch`.

Reply:
I removed the cron